<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profils_manager</class>
 <widget class="QDialog" name="profils_manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="accueilButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Accueil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logOutButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Deconnexion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changeButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="createCB">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>480</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Create profils</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="manageCB">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>480</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Manage profils</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="readCB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="writeCB">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>480</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>520</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>445</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Droits</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>561</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Profils actuels de l'utilisateur</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>531</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>561</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Ajouter un profil à l'utilisateur :</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>701</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Créer un nouveau profil :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
